src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.csproj

```diff
@@ -98,6 +98,8 @@
     <Compile Include="$(CommonTestPath)\System\Collections\IEnumerable.Generic.Serialization.Tests.cs">
       <Link>Common\System\Collections\IEnumerable.Generic.Serialization.Tests.cs</Link>
     </Compile>
+    <Compile Include="ConcurrentBagTests.netstandard1.7.cs" />
```

Comment:
> You will need to add a netcoreapp configuration to correctly build this.

@weshaggard, can you point to an example?  I see, for example, that System.Collections has such new APIs added:
https://github.com/dotnet/corefx/blob/master/src/System.Collections/ref/System.Collections.cs#L60-L63
Which entry in its Configurations.props corresponds to netcoreapp?
https://github.com/dotnet/corefx/blob/master/src/System.Collections/src/Configurations.props
The closest one looks like "netcoreapp1.2corert", but that explicitly talks about corert.